Dockerfile

```diff
@@ -1,12 +1,50 @@
 FROM php:7.3-apache
 
-RUN apt-get update \
- && apt-get install -y git zlib1g-dev libzip-dev \
- && docker-php-ext-install zip \
- && a2enmod rewrite \
- && sed -i 's!/var/www/html!/var/www/public!g' /etc/apache2/sites-available/000-default.conf \
- && mv /var/www/html /var/www/public \
- && curl -sS https://getcomposer.org/installer \
+LABEL maintainer="getlaminas.org" \
+    org.label-schema.docker.dockerfile="/Dockerfile" \
+    org.label-schema.name="Laminas MVC Skeleton" \
+    org.label-schema.url="https://docs.getlaminas.org/mvc/" \
+    org.label-schema.vcs-url="https://github.com/laminas/laminas-mvc-skeleton"
+
+## Update package information
+RUN apt-get update
+
+##  Install zip libraries and extension
+RUN apt-get install --yes git zlib1g-dev libzip-dev
+RUN docker-php-ext-install zip
+
+## Configure Apache
+RUN a2enmod rewrite
+RUN sed -i 's!/var/www/html!/var/www/public!g' /etc/apache2/sites-available/000-default.conf
+RUN mv /var/www/html /var/www/public
+
+## Install Composer
+RUN curl -sS https://getcomposer.org/installer \
   | php -- --install-dir=/usr/local/bin --filename=composer
 
+###
+## Additional PHP extensions.  To use these extensions uncomment the 
+## leading `# ` from the RUN commands.  These are provided to show 
+## examples of including other libraries into the PHP installation.
+###
+
+## Install i18n libraries and extensions
+# RUN apt-get install --yes libicu-dev
+# RUN docker-php-ext-configure intl 
+# RUN docker-php-ext-install intl
+
+## Install mbstring for i18n string support
+# RUN docker-php-ext-install mbstring
+
+## MySQL PDO support
+# RUN docker-php-ext-install pdo_mysql
```

Comment:
I think Postgres support should be added as well and be enabled by default as per comment above

Reply:
> NOTE: the directions in this commit template for refactoring are
> 
> ```
> - Are you refactoring code?
>   - Why do you feel the refactor is necessary?
>   - What types of refactoring are you doing?
>   - TARGET THE develop BRANCH
> ```
> 
> but no develop branch exists on this project (so I'm targeting the only branch: master)

Templates are in a special laminas/.githuib repository and as such they do no account for a couple special cases like this one.

Reply:
If PostgreSQL support in PHP is enabled then a PostgreSQL server should be added via docker-compose.yml  This is leading down a path much more complex than what I started with.  I'm fine with that.  I think a proper Docker configuration supporting all the optional modules would be ideal.

Reply:
If PostgreSQL support in PHP is enabled then a PostgreSQL server should be added via docker-compose.yml This is leading down a path much more complex than what I started with. I'm fine with that. I think a proper Docker configuration supporting all the optional modules would be ideal.

Reply:
> If PostgreSQL support in PHP is enabled then a PostgreSQL server should be added via docker-compose.yml

I disagree. Having support in the image does not mean we have to use it.

> I think a proper Docker configuration supporting all the optional modules would be ideal.

I agree on that. Image should be as easy to use as running composer install with most common packages with no extra hassle. That would be a topic for another PR however.